src/model/transaction/MosaicAddressRestrictionTransaction.ts

```diff
@@ -173,6 +175,22 @@ export class MosaicAddressRestrictionTransaction extends Transaction {
                byteTargetAddress + bytePreviousRestrictionValue + byteNewRestrictionValue;
     }
 
+    /**
+     * Return the string notation for the set recipient
+     * @internal
+     * @returns {string}
+     */
+    public targetAddressToString(): string {
```

Comment:
why name it `target` instead of recipient?

Reply:
this is to match the wg-standard and the catabuffer schema definition